pkg/cmd/api/api_test.go

```diff
@@ -278,6 +278,20 @@ func Test_apiRun(t *testing.T) {
 			stdout: `{"errors": [{"message":"AGAIN"}, {"message":"FINE"}]}`,
 			stderr: "gh: AGAIN\nFINE\n",
 		},
+		{
+			name: "REST string error ",
+			options: ApiOptions{
+				RequestPath: "graphql",
```

Comment:
You can omit `ApiOptions` with `RequestPath` entirely here; it's only necessary for testing GraphQL responses

Reply:
Ok, in that case I will specify an error code 400 since it is a REST operation. It seems to work.